### ISSUE TITLE
bug(select): Two children with the same key, 'test'.

### ISSUE THREAD

**Issue by izakfilmalter** [jamesmfriedman/rmwc#108]
I'm flipping between the css select and the js select with the following code:

```jsx
<Select
    cssOnly={window.matchMedia('(max-width: 767px)').matches}
    ...
/>
```

I get the following error when this switch happens.

```error
index.js:2284 Warning: Encountered two children with the same key, `test`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
    in select (created by SelectSurface)
    in SelectSurface (created by Select)
    in div (created by SelectRoot)
    in SelectRoot (created by Select)
    in Select (created by withMDC(Select))
    in withMDC(Select) (at inputs.jsx:19)
```

**Comment by jamesmfriedman** [comment]
What’s in the ... in your example is important. Do you have the options related to this?

**Comment by izakfilmalter** [comment]
Ya I am working on repro repo again.

**Comment by jamesmfriedman** [comment]
Good usecase on this though. I was planning on doing this in my own app just haven’t gotten around to it. 

The regular Mdc select performance on mobile isn’t great, plus the mobile pickers are better integrated.

**Comment by izakfilmalter** [comment]
Just tested this, fixed by #117.
